offload_vio/device_tx/video_encoder.cpp

```diff
@@ -11,10 +11,7 @@
 #include <thread>
 
 namespace ILLIXR {
-//
-//    void cb_need_data(GstElement *appsrc, guint size, gpointer* user_data) {
-//        reinterpret_cast<video_encoder*>(user_data)->want = 1;
-//    }
+
 // EuRoc
 #define IMG_WIDTH  752
```

Comment:
In hindsight, it is properly easier to have the same structure as the OpenVINs plugin, where you define the data set flag at the beginning, and all the remaining variables are assigned accordingly.

Reply:
Can be done, which will make the change a bit easier :)

Reply:
ok